app/views/products/_actions.html.erb

```diff
@@ -8,9 +8,9 @@
   <td><%= product.discount.percentage if product.discount.present? %>%</td>
   <% if can? :manage, @product %>
     <div class="col">
-      <td><%= link_to 'Edit', edit_product_path(product), class:"btn btn-outline-danger" %></td>
+      <td><%= link_to image_tag("pen-solid.svg", width: 20,float:"right"), edit_product_path(product) %></td>
```

Comment:
Can you change this similar to the _actions.html of stock_statuses?

Reply:
Have made the required changes